src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -286,7 +286,7 @@ public virtual string NormalizeKey(string key)
             ThrowIfDisposed();
             await Store.SetRoleNameAsync(role, name, CancellationToken);
             await UpdateNormalizedRoleNameAsync(role);
-            return await LogResultAsync(IdentityResult.Success, role);
+            return await Task.FromResult(IdentityResult.Success).WithLoggingAsync(this, role);
```

Comment:
Ah, because such version doesn't exist yet for RoleManager... I think it should exist.